### PR TITLE
shimv2: add logging to shimv2 api calls 

### PR DESCRIPTION
**Pull request by snir911**
shimv2: add logging to shimv2 api calls
and also fetch and log container id from the request

Fixes: #2527

### CODE REVIEW

**Review by sameo — APPROVED** [review]

File: src/runtime/containerd-shim-v2/service.go [hunk 1]
```diff
@@ -736,6 +753,8 @@ func (s *service) Resume(ctx context.Context, r *taskAPI.ResumeRequest) (_ *ptyp
 
 // Kill a process with the provided signal
 func (s *service) Kill(ctx context.Context, r *taskAPI.KillRequest) (_ *ptypes.Empty, err error) {
+	shimLog.WithField("container", r.ID).Debug("Kill() start")
+	defer shimLog.WithField("container", r.ID).Debug("Kill end")
```
Comment (liubin) [inline src/runtime/containerd-shim-v2/service.go:757]: Nit: `Kill end` -> `Kill() end`

**Review by liubin — APPROVED** [review]

**Review by fidencio — APPROVED** [review]
lgtm, thanks @snir911!

This is simple enough that, IMHO, deserves to be backported to the `stable-2.2` branch and rolled out together with the `2.2.0` release Tomorrow.

@sameo @liubin @egernst WDYT?

**Pull request merged**